### PR TITLE
fix: typing of columns

### PR DESCRIPTION
**Pull request by OlivierBBB**

### CODE REVIEW

File: logdata/columns.lisp [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  (SIZE_ACC :i32)
-  (SIZE_LIMB :byte)
-  (LIMB :i128)
-  (INDEX :i24))
+  (ABS_LOG_NUM_MAX  :i24)
+  (ABS_LOG_NUM      :i24)
+  (LOGS_DATA        :binary)
+  (SIZE_TOTAL       :i32)
+  (SIZE_ACC         :i32)
+  (SIZE_LIMB        :i5)
```
Comment (OlivierBBB) [inline logdata/columns.lisp:9]: veridise

File: hub/columns/stack.lisp [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
   (;; stack items
-   ( STACK_ITEM_HEIGHT       :array [4])
-   ( STACK_ITEM_VALUE_HI     :array [4])
-   ( STACK_ITEM_VALUE_LO     :array [4])
+   ( STACK_ITEM_HEIGHT       :array [4] :i11)   ;; stack items have heights 0 ≤ h ≤ 1024
+   ( STACK_ITEM_VALUE_HI     :array [4] :i128)
+   ( STACK_ITEM_VALUE_LO     :array [4] :i128)
    ( STACK_ITEM_POP          :array [4] :binary@prove)
-   ( STACK_ITEM_STAMP        :array [4])
+   ( STACK_ITEM_STAMP        :array [4] :i36)   ;; STAMP is :i32 and we consider 8 * STAMP + bla with 0 ≤ bla < 8
```
Comment (OlivierBBB) [inline hub/columns/stack.lisp:12]: type fixes

File: alu/ext/columns.lisp [hunk 3]
```diff
@@ -3,7 +3,7 @@
 (defcolumns
     (STAMP   :i32)
     (OLI     :binary)
-    (CT      :i4)
+    (CT      :i3)
```
Comment (OlivierBBB) [inline alu/ext/columns.lisp:6]: type fix

File: rlptxrcpt/columns.lisp [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
-  (LC_CORRECTION :binary@prove)
-  (PHASE_SIZE :i32)
-  (DEPTH_1 :binary@prove)
-  (IS_TOPIC :binary@prove)
-  (IS_DATA :binary@prove)
-  (LOG_ENTRY_SIZE :i32)
-  (LOCAL_SIZE :i32)
-  (PHASE_ID :i16))
+  (INDEX            :i24)
+  (INDEX_LOCAL      :i24)
```
Comment (OlivierBBB) [inline rlptxrcpt/columns.lisp:12]: type fix

File: shakiradata/columns.lisp [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
-  (IS_SHA2_RESULT :binary@prove)
-  (IS_RIPEMD_DATA :binary@prove)
-  (IS_RIPEMD_RESULT :binary@prove)
+  (SHAKIRA_STAMP          :i32)
+  (ID                     :i32)
+  (PHASE                  :byte)
+  (INDEX                  :i32)
+  (INDEX_MAX              :i32)
+  (LIMB                   :i128)
+  (nBYTES                 :i5)
```
Comment (OlivierBBB) [inline shakiradata/columns.lisp:10]: type fix

**Review by letypequividelespoubelles — APPROVED** [review]

**Comment by letypequividelespoubelles** [comment]
Should regenerate all Trace.java

**Pull request merged**